components/shared/forms/Input/Input.tsx

```diff
@@ -2,10 +2,10 @@ import React from "react";
 import Icon, { IconVariantType } from "../../Icon/Icon";
 
 type Props = {
-  type?: string;
   placeholder: string;
   name: string;
   icon: IconVariantType;
+  type?: string;
```

Comment:
To be honest, you can use `HTMLInputTypeAttribute` type from `react` library. In this case, you will be sure, if you use correct type attr values for this component

Reply:
UPD: you can combine your types(`icon`) with Input attributes `InputHTMLAttributes<HTMLInputElement>`. Check how you can do it https://www.typescripttutorial.net/typescript-tutorial/typescript-intersection-types/